index.js

```diff
@@ -659,10 +660,19 @@ app.post('/smokeTest/teardown', async (request, response) => {
   }
 })
 
-const server = app.listen(8080)
-helpers.log('brave server listening on port 8080')
-if (!helpers.isTestEnvironment()) {
+let server
+
+if (helpers.isTestEnvironment()) {
+  // local http server for testing
+  server = app.listen(8000)
+} else {
+  const httpsOptions = {
+    key: fs.readFileSync(`/etc/brave/ssl/tls.key`),
+    cert: fs.readFileSync(`/etc/brave/ssl/tls.crt`),
```

Comment:
How are these files being put in the right place?

Reply:
These files are being mounted into the pods during the helm deployment. If you look at the changes in sensor-deployment.yaml template, you'll see there's a new volumeMount section that specifies this location, and to put the contents of the sensors-tls secret in it.

Reply:
Cool. Thanks for explaining.